File: backend/micro_epoweredmove_poi_chargingstation/src/main/java/com/kosdiam/epoweredmove/models/POI.java
package com.kosdiam.epoweredmove.models;

import java.io.Serializable;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class POI implements Serializable {
    private String id;
    private Float latitude;
    private Float longitude;
    private Boolean parking;
    private Boolean illumination;
    private Boolean wc;
    private Boolean shopping;
    private Boolean food;
    private String phone;
    private String userId;
    private String chargingStationId;
    private Long timestamp;
}

File: frontend/E-PoweredMoveAndroidApp/app/src/main/java/com/kosdiam/epoweredmove/models/Poi.kt
package com.kosdiam.epoweredmove.models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Poi(
    val id: String,
    val latitude: Float,
    val longitude: Float,
    val parking: Boolean,
    val illumination: Boolean,
    val wc: Boolean,
    val shopping: Boolean,
    val food: Boolean,
    val phone: String,
    val userId: String,
    val userObj: User,
    val chargingStationId: String,
    val chargingStationObj: ChargingStation,
    val paymentMethodsObj: List<PaymentMethod>,
    val timestamp: Long,
    val availableSelectedVehicle: Boolean?,
) : Parcelable
